redis_benchmarks_specification/test-suites/memtier_benchmark-1key-zrevrangebyscore-256K-elements-pipeline-1.yml

```diff
@@ -0,0 +1,37 @@
+version: 0.4
+name: memtier_benchmark-1key-zrevrangebyscore-256K-elements-pipeline-1
+description: 'Runs memtier_benchmark, for a keyspace length of 1 SORTED SET key. The SORTED SET contains 256K elements of 1KB each and we query it using ZREVRANGEBYSCORE.'
+dbconfig:
+  configuration-parameters:
+    save: '""'
+  preload_tool:
+    run_image: redislabs/memtier_benchmark:edge
+    tool: memtier_benchmark
+    arguments: --command="ZADD key1 __key__  __data__" --key-maximum 262144 --key-prefix "" -R --data-size=1024 -n 262144 --hide-histogram -t 1 -c 1
```

Comment:
--command-key-pattern=P

Reply:
@slice4e please include `--command-key-pattern` so this can be approved and merged :)

Reply:
Addressed in #177